h5py/h5.pyx

```diff
@@ -21,6 +21,8 @@ INDEX_NAME      = H5_INDEX_NAME       # Index on names
 INDEX_CRT_ORDER = H5_INDEX_CRT_ORDER  # Index on creation order
 
 HDF5_VERSION_COMPILED_AGAINST = HDF5_VERSION
+NUMPY_VERSION_COMPILED_AGAINST = NUMPY_BUILD_VERSION
+CYTHON_VERSION_COMPILED_AGAINST = CYTHON_BUILD_VERSION
```

Comment:
```suggestion
CYTHON_VERSION_COMPILED_WITH = CYTHON_BUILD_VERSION
```

Small nitpick: I think 'compiled against' means something exposes a C API which we use; Cython is a bit different, in that it's used as part of our build process but we don't end up linked against libcython or something.

Reply:
Done (definitely reads better).